metapool/metapool.py

```diff
@@ -1451,37 +1571,42 @@ def add_syndna(plate_df, syndna_pool_number=None, syndna_concentration=None,
     plate_df_ = plate_df.copy()
```

Comment:
My understanding is trailing _ doesn't mean anything out of the ordinary, it's just a pep8 standard for naming a variable so that it doesn't collide with a reserved word e.g.: set_ instead of set or file_ instead of file. In this case, plate_df is being used as an input parameter and the writer doesn't want to modify it. I've gone ahead and changed the name to 'result'.